SPECS/at/at.spec

```
@@ -174,9 +175,13 @@ chown root:root %{_localstatedir}/spool/at/.SEQ
 %{_bindir}/atrm
 %{_bindir}/atq
 %attr(4755,root,root) %{_bindir}/at
-%attr(0644,root,root) /%{_unitdir}/atd.service
+%attr(0644,root,root) /lib/systemd/system/atd.service
```

nit - generally try to use the %{_libdir} macro instead of /lib

```suggestion
%attr(0644,root,root) /%{_libdir}/systemd/system/atd.service
```